connectivity/lorawan/tests/TESTS/lorawan/loraradio/CMakeLists.txt

```diff
@@ -1,20 +1,22 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-cmake_minimum_required(VERSION 3.19.0 FATAL_ERROR)
+include(mbed_greentea)
 
-set(MBED_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../../../../.. CACHE INTERNAL "")
-set(TEST_TARGET mbed-connectivity-lorawan-loraradio)
+set(SUPPORTED_LABELS ${MBED_TARGET_LABELS})
+list(FILTER SUPPORTED_LABELS INCLUDE REGEX "^(SX1272|SX1276|STM32WL)$")
```

Comment:
Is there any reason for not added `SX126X` in the REGEX check as https://github.com/ARMmbed/mbed-os/blob/master/connectivity/lorawan/tests/TESTS/lorawan/loraradio/main.cpp#L22 have a component header included in the test for COMPONENT_SX126X

Reply:
Good catch. Done.